src/sentry/api/endpoints/organization_relay_keys.py

```diff
@@ -0,0 +1,161 @@
+from __future__ import absolute_import
+
+from copy import deepcopy
+from datetime import datetime
+
+from sentry.api.bases import OrganizationEndpoint, OrganizationAdminPermission
+from sentry.coreapi import APIForbidden
+from sentry import features
+from sentry.models.relay import Relay
+from rest_framework import status
+from rest_framework.response import Response
+from sentry.api.serializers.rest_framework.base import (
+    snake_to_camel_case,
+    camel_to_snake_case,
+    convert_dict_key_case,
+)
+
+ORG_RELAYS_OPTION = "sentry:trusted_relays"
+
+
+class OrganizationRelayKeysEndpoint(OrganizationEndpoint):
+    # TODO (RaduW) decide exactly on the Authorization model that we want
+    # GET=org.read, others org.admin
+    # permission_classes = (OrganizationAuthProviderPermission,)
+    # all HTTP methods org.admin
+    permission_classes = (OrganizationAdminPermission,)
+
+    def check_feature_enabled(self, request, organization):
+        if not features.has("organizations:relay-config", organization, actor=request.user):
+            raise APIForbidden("Relay configuration is not enabled for this organization.")
+
+    def get(self, request, organization):
+        self.check_feature_enabled(request, organization)
+
+        relay_keys = organization.get_option(ORG_RELAYS_OPTION, [])
+        public_keys = [key_info.get("public_key") for key_info in relay_keys]
+        usage = _get_keys_usage(public_keys)
+        ret_val = _add_usage_info(relay_keys, usage)
+        return Response(_to_camel(ret_val), status=status.HTTP_200_OK)
```

Comment:
Generally we would use rest_framework serializers, would that be something we could do here? It would give you a place to put your _ prefixed functions.

Reply:
@RaduW I believe you have some feedbacks 😉